packages/@glimmer/debug-util/lib/present.ts

```diff
@@ -14,8 +14,8 @@ export function assertPresent<T>(value: T, message?: string): asserts value is P
   }
 }
 
-export function isPresentArray<T>(list: readonly T[]): list is PresentArray<T> {
-  return list.length > 0;
+export function isPresentArray<T>(list?: readonly T[]): list is PresentArray<T> {
```

Comment:
Hm... what uses `isPresentArray` with an optional array?

Reply:
nothing -- it's a minification bug!

input:
```ts
  private matchFor(
    left: OffsetKind,
    right: OffsetKind
  ): (left: PositionData, right: PositionData) => Out {
    const nesteds = this._whens.match(left);

    localAssert(
      isPresentArray(nesteds),
      `no match defined for (${left}, ${right}) and no AnyMatch defined either`
    );

    const callback = new WhenList(nesteds).first(right);

    localAssert(
      callback !== null,
      `no match defined for (${left}, ${right}) and no AnyMatch defined either`
    );

    return callback;
  }
```

output: 
```js
 matchFor(t, e) {
    const s = this._whens.match(t);
    return P(), new I(s).first(e);
  }
```